NDTensors/src/lib/TypeParameterAccessors/src/Base/iswrappedarray.jl

```diff
@@ -0,0 +1,71 @@
+using SimpleTraits
+using LinearAlgebra:
+  Adjoint,
+  Diagonal,
+  Hermitian,
+  LowerTriangular,
+  Symmetric,
+  Transpose,
+  UnitLowerTriangular,
+  UnitUpperTriangular,
+  UpperTriangular
+using Base: ReshapedArray, SubArray
+using StridedViews: StridedView
+# Trait indicating if the AbstractArray type is an array wrapper.
+# Assumes that it implements `NDTensors.parenttype`.
+@traitdef IsWrappedArray{ArrayT}
+
+#! format: off
+@traitimpl IsWrappedArray{ArrayT} <- is_wrapped_array(ArrayT)
+#! format: on
+
+"""
+Optional definitions for types which are considered `Wrappers` and have a `parenttype`
+
+  Should return a `Position`.
+"""
+function parenttype_position(type::Type)
+  return 0
+end
+
+for wrap in (
```

Comment:
```suggestion
for wrapper in (
```

Reply:
I just think that sounds better, `wrap` is a verb so it makes it seem like it will be doing something, while `wrapper` is a noun.

Reply:
Sounds good I have updated to `wrapper`